### ISSUE TITLE
Provide a public accessibility statement that reflects our policy, goals, and achievements.

### ISSUE THREAD

**Issue by joshdanielson** [loconomics/loconomics#516]
ACME Inc. is committed to ensuring that its website is accessible to people with disabilities. All the pages on our website will meet W3C WAI's Web Content Accessibility Guidelines 2.0, Level AA conformance. Any issues should be reported to accessibility@acme.inc.

**Comment by joshdanielson** [comment]
Included in terms/accessibility-policy. Closing
